gerrit/install.sls

```diff
@@ -24,12 +24,36 @@ group_{{ settings.group }}:
     - group: {{ settings.group }}
     - makedirs: true
 
+{{ settings.base_directory }}/{{ settings.site_directory }}/lib:
+  file.directory:
+    - user: {{ settings.user }}
+    - group: {{ settings.group }}
+    - makedirs: true
+
+{% for name, library in salt['pillar.get']('gerrit:libraries', {}).items() %}
+{{ install_dir }}/{{ site_dir }}/lib/{{ name }}.jar:
+  file.managed:
+    - source: {{ library.source }}
+    - source_hash: {{ library.source_hash }}
+    - user: {{ user }}
+    - group: {{ group }}
+{% endfor %}
+
 {{ settings.base_directory }}/{{ settings.site_directory }}/plugins:
   file.directory:
     - user: {{ settings.user }}
     - group: {{ settings.group }}
     - makedirs: true
 
+{% for name, plugin in salt['pillar.get']('gerrit:plugins', {}).items() %}
```

Comment:
There's no pillar file for the gerrit formula and I don't see the plugins defined anywhere else other than the directory creation. Could you please add a `pillar.example` that provides users with an example of specifying their plugins?